webpages/settings/index.js

```diff
@@ -135,34 +177,32 @@ const AddonBody = Vue.extend({
       return this.$root.addonSettings;
     },
     addonMatchesFilters() {
+      if (this.groupId !== "search" && this.searchInput !== "") return false;
       if (!this.addon._wasEverEnabled) this.addon._wasEverEnabled = this.addon._enabled;
 
-      const matchesTag = this.$root.selectedTag === null || this.addon.tags.includes(this.$root.selectedTag);
       const matchesSearch =
         this.searchInput === "" ||
         this.addon.name.toLowerCase().includes(this.searchInput.toLowerCase()) ||
         this.addon._addonId.toLowerCase().includes(this.searchInput.toLowerCase()) ||
-        this.addon.description.toLowerCase().includes(this.searchInput.toLowerCase()) ||
+        this.addon.description?.toLowerCase().includes(this.searchInput.toLowerCase()) ||
+        this.addon.versionAdded?.includes(this.searchInput) ||
```

Comment:
Why `this.addon.description?.`? Description is mandatory
Also, we should probably find a better way to search by versionAdded later, instead of doing this

Reply:
I must of got an error somewhere, but should be safe to remove from now.